src/redux/reducers/index.js

```diff
@@ -1,18 +1,23 @@
 import { combineReducers } from 'redux';
```

Comment:
Yes thanks for bringing it up! I agree, we should separate into files and use index to import all of them. I was unsure how to proceed with folder organization so I just left it in index. Either someone can merge the request and re-organize files or I can do it and just update this PR.